instrumentation/opentelemetry-instrumentation-aws-lambda/tests/test_aws_lambda_instrumentation_manual.py

```diff
@@ -264,6 +279,7 @@ def custom_event_context_extractor(lambda_event):
             ),
         ]
         for test in tests:
+            print(f"Executing test: {test.name}")
```

Comment:
please use self.subTest https://docs.python.org/3/library/unittest.html#distinguishing-test-iterations-using-subtests

Reply:
@xrmx good call, the `print` statement was originally added because it was difficult to nail down exactly which sub-test was failing. I've incorporated your feedback and now each sub-test is wrapped in a `subTest` context manager.